### DISCUSSION TITLE
5166 accidental crash

### DISCUSSION THREAD

**Discussion by whxuezhe** [Alex313031/Thorium-Win-AVX2#19] [Q&A]
When I was by 70kankan When com points to Bing, the browser closes unexpectedly; It also did not trigger the recovery operation of restarting the browser due to an unexpected crash. Please test the location repair, thank you!

**Comment by gz83** [comment]
> When I was by 70kankan When com points to Bing, the browser closes unexpectedly; It also did not trigger the recovery operation of restarting the browser due to an unexpected crash. Please test the location repair, thank you!

@whxuezhe Please pay attention to the recent outflow about the impact of the 12th generation Intel processor core display on the performance and operation of the chrome series browsers. You often encounter crashes when using Thorium, which may be related to the nuclear display driver. In addition, please use the AVX version for testing. The last time has given you guidelines.

**Reply by whxuezhe** [reply]
@gz83 Only 30% of your words are on the point. In the early stage, I was indeed the Intel Core Graphics UHD770 31.0.101.1999, but in the next few days, Microsoft Update forced back to the core graphics driver of the OEM version 30.0.101.1369, which can only be Another way to judge the performance or efficiency of the 12th generation core is that it does not support the AVX2 instruction set very well, not as good as the 11th generation.

**Reply by gz83** [reply]
> @gz83 Only 30% of your words are on the point. In the early stage, I was indeed the Intel Core Graphics UHD770 31.0.101.1999, but in the next few days, Microsoft Update forced back to the core graphics driver of the OEM version 30.0.101.1369, which can only be Another way to judge the performance or efficiency of the 12th generation core is that it does not support the AVX2 instruction set very well, not as good as the 11th generation.

@whxuezhe Didn't I keep telling you to use the AVX version to test? And I told you earlier about the AVX2 instruction set of the 12th generation processor. Of course, it cannot be ruled out that the nuclear display driver has a certain influence.
